pkgs/tools/misc/opentelemetry-collector/contrib.nix

```diff
@@ -41,16 +42,21 @@ buildGoModule rec {
 
   preCheck = "export CGO_ENABLED=1";
 
-  # This test fails on darwin for mysterious reasons.
-  checkFlags = lib.optionals stdenv.isDarwin
-    [ "-skip" "TestDefaultExtensions/memory_ballast" ];
+  checkFlags = [
+    # collectd test tries and fails to bind to port 8081
+    # mezo test tries to reach external service
+    # memory_ballast test fails on darwin for mysterious reasons.
+    "-skip=TestDefaultReceivers/collectd|TestDefaultExporters/mezmo${if stdenv.isDarwin then "|TestDefaultExtensions/memory_ballast" else ""}"
+  ];
 
   ldflags = [
     "-s"
     "-w"
     "-X github.com/open-telemetry/opentelemetry-collector-contrib/internal/version.Version=v${version}"
   ];
 
+  passthru.updateScript = nix-update-script { };
```

Comment:
Personally I don't put these in when it's just the basic updater as r-ryantm manages just fine and using `nix-update` myself is pretty easy. I don't see a massive amount of people using the maintainer script to trigger these.

But I don't mind that much

Reply:
Ah TIL, I didn't know that. I guess it just wasn't creating the PRs automatically because the tests were failing